Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use an official golang image for simplicity (and compilation)
+FROM golang:1.9.7-stretch
```

Comment:
Shouldn't we use the lastest version of go? Is there any incompatibility with Swarm?

Reply:
Yeah, no reason not to. I just missed that there was a newer version of go out.